src/components/TodoModal/useUser.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+export const useUser = (userId: number) => {
```

Comment:
We might move this custom hook to more generic directory. It should not be inside TodoModal